src/components/SideNav/_side-nav-multi-tier.scss

```diff
@@ -0,0 +1,45 @@
+$textColor: #f3f3f3;
```

Comment:
Are there no carbon variables for these colors?

Reply:
#f3f3f3 sounds like the old Gray 10 color, which is now #f4f4f4.

The equivalent color token name for the text in the Gray 10 theme is $inverse-01 though I believe (that's what I have on my spec)

Reply:
yes I have updated textColor to be the `$inverse-01`